docs/Configuring.md

```diff
@@ -173,6 +173,39 @@ database {
 }
 ```
 
+**Cromwell server on MySQL Database**
+
+Uses docker-compose to link together a Cromwell docker image (built locally with `sbt docker` or available on [dockerhub](https://hub.docker.com/r/broadinstitute/cromwell/)) with a MySQL docker image.
+To change the version of Cromwell used, change the tag in `compose/cromwell/Dockerfile`
+
+**Local**
+
+`docker-compose up` from this directory will start a Cromwell server running on a mysql instance with local backend.
+The default configuration file used can be found at `compose/cromwell/app-config/application.conf`
+To override it, simply mount a volume containing your custom `application.conf` to `/app-config` (see `jes-cromwell/docker-compose.yml` for an example)
+
+**JES**
+
+The `jes-cromwell` directory is an example of how to customize the original compose file with a configuration file and environment variables.
```

Comment:
As this doc has moved from living beside the mentioned scripts, I'd put a link(s) back. This could be a single link to the root https://github.com/broadinstitute/cromwell/tree/develop/scripts, or sprinkled direct links throughout these additional sections to directories like https://github.com/broadinstitute/cromwell/tree/develop/scripts/docker-compose-mysql/jes-cromwell.